scripts/setup-ubuntu.sh

```diff
@@ -70,6 +71,15 @@ PACKAGES+=" bsdmainutils"
 # Needed by package ccnet.
 PACKAGES+=" valac"
 
+# Needed by package libgcrypt.
+PACKAGES+=" fig2dev"
+
+# Needed by package libidn2.
+PACKAGES+=" gengetopt"
+
+# Needed by package proxmark3-git.
+PACKAGES+=" swig"
+
```

Comment:
Could you give example of the issue which happens when these packages are not installed?

Reply:
These programs are called when building these packages and result in binary not found, i fixed by `sudo apt install ...`  and then update this script, I already noted the requiring package.